src/gluonts/support/util.py

```diff
@@ -175,3 +175,241 @@ def erfinv(x: np.array) -> np.array:
         p = c + p * w
 
     return p * x
+
+
+class Interpolation:
+    """
+    Interpolation based on trained quantile predictions
+
+    Parameters
+    ----------
+    quantile_predictions
+        Dict with key = str(quantile) and value is np.array with size prediction_length
+    interpolation_type
+
+    """
+
+    def __init__(
+        self, quantile_predictions: dict, interpolation_type: str = "linear"
```

Comment:
Is it necessary to tie this class to the “quantile” naming? Can’t this be used for general interpolation?

Reply:
Yes, I think @youngsuk0723's implementation here was doing interpolation on the quantile predictions, where the knots are the quantiles but this can be generalized.